docs/api.md

```diff
@@ -4,10 +4,18 @@ Popover attaches to an element and displays on hover/blur.
 
 ## Properties
 
-| Property    | Attribute   | Type     | Default | Description                                      |
-|-------------|-------------|----------|---------|--------------------------------------------------|
-| `for`       | `for`       | `String` |         | Defines an `id` for an element in the DOM to trigger on hover/blur. |
-| `placement` | `placement` | `String` | "top"   | Expects top/bottom - position for popover in relation to the element. |
+| Property           | Attribute          | Type      | Default | Description                                      |
+|--------------------|--------------------|-----------|---------|--------------------------------------------------|
+| `for`              | `for`              | `String`  |         | Defines an `id` for an element in the DOM to trigger on hover/blur. |
+| `isPopoverVisible` | `isPopoverVisible` | `boolean` | false   | Boolean for if popover is visible or not.        |
+| `isSticky`         | `isSticky`         | `boolean` |         | If true, when user clicks trigger, the popover will persist its visibility. If false, popover will disappear when mouseout over trigger. |
```

Comment:
This is really confusing. The actual API property is `sticky`. That is what the user will add to the element for this feature. 

The `isSticky` is internal.

Reply:
Changed.